Rakefile

```diff
@@ -347,13 +347,13 @@ end
 task :setup => ["git:clone", "bundle:install"]
 
 task :default do
-  run_command "bin/rake"
+  run_command "bundle exec rake"
 end
 
 desc "publish cukes to relishapp.com"
 task :relish, :version do |_, args|
   raise "rake relish[VERSION]" unless args[:version]
-  run_command "bin/rake relish['#{args[:version]}']", :except => ['rspec']
+  run_command "bundle exec rake relish['#{args[:version]}']", :except => ['rspec']
```

Comment:
Yes, the binstubs create by Bundler rely on shebangs to run so they are Linux/Mac only.  There is a gem that will produce Windows-compatible stubs but I was reluctant to add a gem dependency when there was a simple, cross platform, solution.

I guess one could add `RAKE_COMMAND` as you suggest but it just seems like additional complexity.

One could still use `--binstubs` and `bin/rake`, `bin/yard`, etc. from the command line.  Only in scripts would it need to change.